storage/api.go

```diff
@@ -83,6 +83,9 @@ type ConsensusSourceStorage interface {
 	// GenesisDocument returns the genesis document for the chain.
 	GenesisDocument(ctx context.Context) (*genesisAPI.Document, error)
 
+	// AtHeightData returns all data tied to a specific height.
+	AtHeightData(ctx context.Context, height int64) (*ConsensusAtHeightData, error)
```

Comment:
(totally non-blocking nit/thought) I was thinking `DataAtHeight` rolls off the tongue or `AllData`.

Reply:
It does sound weird. Changed to `AllData`.